app/src/main/java/com/ustwo/boilerplate/base/BasePresenter.java

```diff
@@ -41,7 +40,6 @@ public void onViewWillShow(@NonNull final T view) {
      */
     @CallSuper
     public void onViewWillHide() {
```

Comment:
sometimes there's need to do some _clean up_ to views so it might be a good a keep the disposables, unless we accept we could access it the each Activity/View feature and deal with it there.